android/sdk/src/main/java/org/jitsi/meet/sdk/ConnectionService.java

```diff
@@ -448,5 +455,47 @@ public String toString() {
                     "ConnectionImpl[address=%s, uuid=%s]@%d",
                     getAddress(), getCallUUID(), hashCode());
         }
+
+        /**
+         * Helper function to send an event to JavaScript.
+         *
+         * @param eventName {@code String} containing the event name.
+         * @param data {@code Object} optional ancillary data for the event.
+         */
+        public void emitEvent(
+            String eventName,
+            @Nullable Object data) {
+            ReactApplication rnApp = (ReactApplication) getApplication().getApplicationContext();
```

Comment:
We don't use ReactApplication so this won't work I'm afraid.

Reply:
Updated.